Sources/Interaction/Style/InteractorStyleManipulator/index.js

```diff
@@ -427,6 +432,15 @@ function vtkInteractorStyleManipulator(publicAPI, model) {
     }
   };
 
+  //-------------------------------------------------------------------------
+  publicAPI.ignoreButtonUp = (button) =>
+    button === LEFT_MOUSE_BUTTON && model._interactor.isPointerLocked();
+
+  //-------------------------------------------------------------------------
+  publicAPI.handleEndPointerLock = () => {
+    model.currentManipulator = null;
```

Comment:
shouldn't there be an `endInteraction()` triggered as well as `cancelAnimation` ...?

Reply:
Yes, I think that `currentManipulator.onButtonUp()` should be called in `onButtonUp` even in pointerLock mode.